dokan/dokan.c

```diff
@@ -369,7 +437,7 @@ UINT WINAPI DokanLoop(PDOKAN_INSTANCE DokanInstance) {
     if (!status) {
       lastError = GetLastError();
       DbgPrint("Ioctl failed for wait with code %d.\n", lastError);
-      if (lastError == ERROR_NO_SYSTEM_RESOURCES) {
+      if (lastError == ERROR_DEV_NOT_EXIST) {
```

Comment:
This if (lastError == ERROR_NO_SYSTEM_RESOURCES)

Should remain. ERROR_DEV_NOT_EXIST is wrong here